src/datastore/postgres/repos/utils.ts

```diff
@@ -0,0 +1,19 @@
+import { TypeORMError } from 'typeorm';
```

Comment:
if we don't want to add test cases for this file, the we should add istanbul ignore here as well